scripts/check_version

```diff
@@ -1,17 +1,27 @@
 #!/bin/bash
 
-SRC_VERSION=`grep "const VERSION " ./src/constants.js | cut -d"'" -f2`
+set -e  # Exit on any command failure.
+set -u  # Exit on unset variables.
+
 PACKAGE_VERSION=`grep '"version":' ./package.json | cut -d '"' -f4`
```

Comment:
Why not also use the node command here?

Reply:
We can transform this into a common action later, centralizing the code for the version check for all our NodeJs applications.